index.html

```diff
@@ -0,0 +1,1591 @@
+                    </li>
+                    <li class="latest__item deal__iteam"><span>$0</span></li>
+                  </ul>
+                </div>
+              </li>
+              <!-- End Latest Deal Artile Block -->
+
+              <!-- Start Latest Deal Artile Block -->
+              <li class="col-m latest__li">
+                <a href="#" class="latest__img"></a>
+                <div class="latest__box">
+                  <h4 class="latest__heading--second forth-heading">
+                    <a href="#">
+                      Instagram now lets you pin 3 comments on your posts -
+                      here's how
+                    </a>
+                  </h4>
+                  <ul class="latest__list">
+                    <li class="latest__item deal__iteam">
+                      <span class="deal__original">$49.99</span>
+                    </li>
+                    <li class="latest__item deal__iteam"><span>$0</span></li>
+                  </ul>
+                </div>
+              </li>
+              <!-- End Latest Deal Artile Block -->
+
+              <!-- Start Latest Deal Artile Block -->
+              <li class="col-m latest__li">
+                <a href="#" class="latest__img"></a>
+                <div class="latest__box">
+                  <h4 class="latest__heading--second forth-heading">
+                    <a href="#">
+                      Instagram now lets you pin 3 comments on your posts -
+                      here's how
+                    </a>
+                  </h4>
+                  <ul class="latest__list">
+                    <li class="latest__item deal__iteam">
+                      <span class="deal__original">$49.99</span>
+                    </li>
+                    <li class="latest__item deal__iteam"><span>$0</span></li>
+                  </ul>
+                </div>
+              </li>
+              <!-- End Latest Deal Artile Block -->
+
+              <!-- Start Latest Deal Artile Block -->
+              <li class="col-m latest__li">
+                <a href="#" class="latest__img"></a>
+                <div class="latest__box">
+                  <h4 class="latest__heading--second forth-heading">
+                    <a href="#">
+                      Instagram now lets you pin 3 comments on your posts -
+                      here's how
+                    </a>
+                  </h4>
+                  <ul class="latest__list">
+                    <li class="latest__item deal__iteam">
+                      <span class="deal__original">$49.99</span>
+                    </li>
+                    <li class="latest__item deal__iteam"><span>$0</span></li>
+                  </ul>
+                </div>
+              </li>
+              <!-- End Latest Deal Artile Block -->
+            </ul>
+          </section>
+        </div>
+      </div>
+    </main>
+
+    <!-- Start Footer -->
+    <footer class="footer">
+      <div class="link-box">
```

Comment:
The footer should not be changing its style throughout the breakpoints. It should be maintaining the same style in all screen sizes.

| Source Page	| Clone Page |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/57726348/87092793-b8692d80-c259-11ea-9321-e0c4153b8357.png)|![Annotation 2020-07-10 025946](https://user-images.githubusercontent.com/57726348/87092808-bf903b80-c259-11ea-8425-a47d0b2c80bf.jpg)|